node/res/chain_specs/litentry.json

```diff
@@ -16,7 +16,7 @@
   "protocolId": "litentry",
   "properties": {
     "ss58Format": 31,
-    "tokenDecimals": 12,
+    "tokenDecimals": 18,
```

Comment:
I have applied change to your reference.
I do not know. It might effect if the frontend use chain_spec for displaying purposes.
I simply change as much deicmal 12 related impl as possible.